osu.Game/Skinning/LegacySkinExtensions.cs

```diff
@@ -72,7 +72,7 @@ protected override void LoadComplete()
                 if (timeReference != null)
                 {
                     Clock = timeReference.Clock;
-                    PlaybackPosition = timeReference.AnimationStartTime - timeReference.Clock.CurrentTime;
+                    PlaybackPosition = timeReference.Clock.CurrentTime - timeReference.AnimationStartTime;
```

Comment:
@smoogipoo requesting you sanity check this – it was back-to-front before

Reply:
I think this is correct, if the clock is 100ms behind the animation start, this will be -100, and will take 100ms to elapse before the animation starts.